packages/common/src/components/TableView/ManageColumnsToolbar.tsx

```diff
@@ -32,6 +31,7 @@ export interface ManageColumnsToolbarProps {
   defaultColumns: ResourceField[];
   /** Setter to modify state in the parent.*/
   setColumns(resourceFields: ResourceField[]): void;
+  labels: Record<keyof typeof defaultLabels, string>;
```

Comment:
refactor path can be sending the labels using the `ResourceField[]` data

Reply:
@yaacov  
Could you provide some example how it should work?